src/server/routes.ts

```diff
@@ -1,9 +1,20 @@
 import express from 'express';
+import fetchSingleStockbyTime from './fmpSearch';
 
 const router = express.Router();
 
-router.get('/crypto', (_req, res) => {
-  res.status(404).send('Hello World');
+router.get('/singleStock', async (_req, res, next) => {
+  try {
+    const { query } = _req;
+    if (typeof query.stock !== 'string' || typeof query.time !== 'string') {
+      res.status(404).send('Hello World');
```

Comment:
Hello World?

Reply:
Was still there from testing. Changed it now.